### ISSUE TITLE
Use ContentSync from Sketch Scripting to get Content Versions

### ISSUE THREAD

**Issue by dbrody** [contentsync/SketchContentSync#126]
# Request
Feature request by user to get ContentSync versions from a run script or other plugin code.

# Proposed Solution

With the Sketch ContentSync plugin installed, on the run script of sketch you can now do something like this:
```javascript
function withContentSyncVersion(version){
  console.log(version);
}

ContentSync.API().versions((ok, data) => {
  if(ok) data.map(withContentSyncVersion);
  if(!ok) console.log("Error: " + data);
});
```

The return value of `data` on `ok=true` is an array of ContentSync versions on the active project.

Each version is of shape:
```json
{
  "name": <string>,
  "syncable": <bool>
}
```

# Example

<img width="626" alt="Screen Shot 2021-07-15 at 11 05 37 PM" src="https://user-images.githubusercontent.com/210322/125900145-7fe5d10f-2721-4bb8-a572-898f76c4b869.png">

**Comment by dbrody** [comment]
This initial version has been added as of 8.3.0.

**Comment by nilsdoehring** [comment]
Thank you so much @dbrody for adding this, very helpful! And apologies for not replying earlier. May I ask if this is the single API command available in 8.3.0? Based on the data returned by the _versions_ command, I'd like to do:
```
for each (d in data){
  if(d.syncable){
    //omitted: duplicate a page in Sketch, name it by the value of d.name, and select all objects on that page
    ContentSync.API().syncSelection(d.name, selection);
  }
}
```
